assets/scripts/dialogs/Analytics/MiniGraph.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  XYPlot,
+  XAxis,
+  YAxis,
+  VerticalGridLines,
+  HorizontalGridLines,
+  VerticalBarSeries
+} from 'react-vis'
+
+const totalCount = (arr, dataKeys) => {
+  if (!arr || !arr.length) return
+  const sums = {}
+
+  Array.prototype.forEach(arr, function (item) {
+    Array.prototype.forEach(dataKeys, function (option) {
+      sums[option] = (sums[option] || 0) + item[option]
+    })
+  })
+  return Object.entries(sums).map(item => {
+    const [ key, value ] = item
+    return { x: key, y: value }
+  })
+}
+
+export default class Example extends React.Component {
+  static propTypes = {
+    width: PropTypes.number,
+    height: PropTypes.number,
+    data: PropTypes.array,
+    dataKeys: PropTypes.array
+  }
+
+  render () {
+    const { data, dataKeys, width, height } = this.props
+    const transform = (dataArray) => {
+      const ans = totalCount(dataArray, dataKeys)
+      return ans
+    }
+
+    return (
+      <div>
```

Comment:
What does the `<div>` do for us here?

Reply:
I'm actually not using this component in the current version, so I'm just gonna cut it and re-introduce a better version as we iterate. ✂️